addons/stock/models/stock_picking.py

```diff
@@ -438,6 +438,9 @@ def _get_action(self, action_xmlid):
                     'default_company_id': allowed_company_ids[0],
                 })
 
+        if self.code == 'incoming':
+            context.update({'is_receipt': True})
+
```

Comment:
Put it in the xml action directly

Reply:
@amoyaux  No, we can't cause there is no dedicated action for receipt or any operation types when opening from an overview.
https://github.com/odoo/odoo/blob/0c479082f16a3c04bad754cbeb1abf56f42d5b3d/addons/stock/models/stock_picking.py#L468

Reply:
> @amoyaux No, we can't **cause there is no dedicated action for receipt** or any operation types when opening from an overview.

I think `action_picking_tree_incoming`  (and its siblings) disagrees with you  :suspect: